docs/hass-init.md

```diff
@@ -24,19 +24,6 @@ You are starting with the creation of the owner account of Home Assistant. This
 
 4. Finally, you will see the Home Assistant web interface, which will show all of your devices. 
 
-## Next step
-
-Next you need to setup MQTT Broker using one of the options:
-
-**Option 1 (Home Assistant OS)**
-* If you are upgrading your **Home Assistant OS**, [go here](/docs/mqtt-hassos/).
-
-**Option 2 (Pre-installed image or Home Assistant Docker or Home Assistant Core)**
-* If you are using **pre-installed image** or upgrading your **Home Assistant Docker** or **Core**, [go here](/docs/mqtt-image-docker-core/).
-
-## Related videos
-
-https://youtu.be/n4PX25aB1JU
 
 ## Troubleshooting
 
```

Comment:
On this step we haven't set up the Robonomics Integration yet. It might be better to move the information about password recovery to the Global Administration page.

Reply:
fixed https://github.com/airalab/robonomics-wiki/pull/632/commits/7162715d12486f2d2066492cd74dc2e3ea69dc02